test/Core.Test/Utilities/SpanExtensionsTests.cs

```diff
@@ -15,18 +18,18 @@ public class SpanExtensionsTests
     [InlineData("T.T.T", "T", "T.T")]
     public void TrySplitBy_CanSplit_Success(string fullString, string firstPart, string secondPart)
     {
-        var success = fullString.AsSpan().TrySplitBy('.', out var firstPartSpan, out var secondPartSpan);
+        var success = fullString.AsSpan().TrySplitBy(_periodChar, out var firstPartSpan, out var secondPartSpan);
         Assert.True(success);
         Assert.Equal(firstPart, firstPartSpan.ToString());
         Assert.Equal(secondPart, secondPartSpan.ToString());
     }
 
     [Theory]
-    [InlineData("Test", '.')]
-    [InlineData("Other test", 'S')]
-    public void TrySplitBy_CanNotSplit_Success(string fullString, char splitChar)
+    [InlineData("Test", ".")]
+    [InlineData("Other test", "S")]
+    public void TrySplitBy_CanNotSplit_Success(string fullString, string splitChar)
```

Comment:
❓ I see this moved to `string` but I don't quite see why based on the .NET docs.

Reply:
Because I use it in the `SearchValues.Create` method, which only takes string or bytes.